app/models/program.rb

```diff
@@ -49,7 +51,7 @@ def confirmed
 #   validates :conference_id, presence: true, uniqueness: true
   validates :rating, numericality: { greater_than_or_equal_to: 0, less_than_or_equal_to: 10 }
 
-  before_create :create_event_types
+  after_create :create_event_types
```

Comment:
What about the next line? Don't we have the same issue with difficulty levels?
https://github.com/openSUSE/osem/blob/master/app/models/program.rb#L53